docs/source/create/avatars/create-avatars.rst

```diff
@@ -38,17 +38,11 @@ The steps involved in creating your avatar are:
 3. Fine tune your avatar using a tool such as Blender or Maya.
 4. Package the model in Vircadia for use as an avatar.
 
-Check out this YouTube playlist for one way to create and customize your own avatar. Here, we use Mixamo to rig our avatar automatically, and Blender to adjust the rendering on our avatar. We also have written instructions on the same process:
-
 * :doc:`Rig Your Avatar in Mixamo <mixamo-tutorial>`
 * :doc:`Modify Materials and Textures with Blender <blender-tutorial>`
 
 .. raw:: html
```

Comment:
As Sphinx indicates with a warning: The HTML that you removed under this is actually the content clock for `.. raw:: html`, so you need to remove that as well.

Reply:
I wasn't aware that Sphinx indicated that to me.